source/statements.tex

```diff
@@ -694,14 +694,30 @@
 \end{note}
 \end{itemize}
 \end{itemize}
-
 \begin{example}
 \begin{codeblock}
 int array[5] = { 1, 2, 3, 4, 5 };
 for (int& x : array)
   x *= 2;
 \end{codeblock}
 \end{example}
+\begin{note}
```

Comment:
The paper instructs
> In subclause 8.6.5 [stmt.ranged] add before Example 1:

This was added _after_ Example 1.

Reply:
See commit message.